skin/light/light.css

```diff
@@ -178,6 +181,30 @@
   color: #333 !important;
 }
 
+.tabbrowser-tab.tab-hidden {
+  animation: var(--tab-animation-time) fade-out, var(--tab-animation-time) var(--tab-animation-time) slide-out;
+}
+
+.tabbrowser-tab.tab-visible {
+  animation: 1s slide-fade-in;
```

Comment:
This is _really_ slow…  Can we make this `--tab-animation-time`, and define that to be `300ms` above?